src/PowerShell/Microsoft.WinGet.Client.Engine/Helpers/AppxModuleHelper.cs

```diff
@@ -63,6 +67,9 @@ internal class AppxModuleHelper
 
         // Assets
         private const string MsixBundleName = "Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle";
+        private const string DependenciesAssetName = "DesktopAppInstaller_Dependencies";

```

Comment:
Why is this needed?  A search on the PR didn't find it being used.

Reply:
That was meant to be the name of the extracted zip folder which I included in the asset full path.